services/ingestion/subscriber.go

```diff
@@ -33,5 +34,10 @@ func (r *RPCSubscriber) Subscribe(ctx context.Context, height uint64) (<-chan fl
 		},
 	}
 
-	return r.client.SubscribeEventsByBlockHeight(ctx, height, filter)
+	evs, errs, err := r.client.SubscribeEventsByBlockHeight(ctx, height, filter)
```

Comment:
```suggestion
	evs, errs, err := r.client.SubscribeEventsByBlockHeight(ctx, height, filter, grpc.WithHeartbeatInterval(1))
```

We actually need to use the above variant, but this is only available for this client:
```go
client, err := grpc.NewBaseClient(
	config.AccessURL,
	goGrpc.WithTransportCredentials(insecure.NewCredentials()),
)
```
If it's too much trouble to add it now, we should add a TODO and tackle it later.

Reply:
I believe the default heartbeat is 100, which is working well for now, but I will add todo if we should revisit it.